### DISCUSSION TITLE
Keypad: how to get information in MQTT that the back-key has been pressed

### DISCUSSION THREAD

**Discussion by staudtm** [technyon/nuki_hub#520] [Q&A]
Hello,
I am totally new with Nuki Hub. I switched off the internal Wifi+MQTT of my Nuki Smart Lock 4.0 Pro and I am using now Nuki Hub (and MQTT).
The challenge is to get somehow the information that Smart Lock was locked by pressing the BACK-KEY of my Keypad.
Is it possible to get this information via MQTT?
Regards
Markus

**Comment by iranl** [comment]
Set the Nuki device admin PIN in the NukiHub credentials and enable the "Publish authorization log" option in NukiHub access control configuration.

You should then get multiple log topics in MQTT where the rollingLog topic has the most recent authorization log with information of the origin of the lock action. You should be able to discern if the back key was used to lock from this info.

**Comment by staudtm** [comment]
Thanks for the fast response. I found "rollingLog_trigger eq arrowkey" which should work. Question: I can see topics named

"log_x_action", log_x_authorizationId", ...
and
 "shortlog_x_action", shortlog_x_authorizationId", ...

What is the purpose/difference?

**Comment by iranl** [comment]
This is all covered in the [README](https://github.com/technyon/nuki_hub)

**Comment by staudtm** [comment]
Sorry, you are right. My mistake.

**Comment by staudtm** [comment]
Got all the information I need
